aiokafka/coordinator/assignors/sticky/sticky_assignor.py

```diff
@@ -498,10 +557,9 @@ def _perform_reassignments(self, reassignable_partitions):
                     )
                 # the partition must have a current consumer
                 consumer = self.current_partition_consumer.get(partition)
-                if consumer is None:
-                    log.error(
-                        "Expected partition %r to be assigned to a consumer", partition
-                    )
+                assert (
+                    consumer is not None
+                ), f"Expected partition {partition!r} to be assigned to a consumer"
```

Comment:
Sorry for changing my mind. Looks like my suggestion may potentially kill coordination task. Let's return to your variant, but reword it this way:
```suggestion
                if consumer is None:
                    log.error(
                        "Expected partition %r to be assigned to a consumer", partition
                    )
                    continue
```
Please recheck that its behaviour is equivalent to what we had before.